aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,108 @@
+
+> [!div class="checklist"]
+> * Create a SignalR on ASP.NET Core web app.
+> * Create a SignalR hub to push content to clients.
+> * Use the SignalR JavaScript library to send messages and display updates from the hub.
+
+# Prerequisites
+
+Install the following:
+
+* [.NET Core 2.1.0 Preview 1 SDK](https://www.microsoft.com/net/core) or later.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
+
+## Create an ASP.NET Core project that hosts SignalR client and server
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**.
+1. Name the project `SignalRChat`
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
+
+1. Select **Web Application** which creates a project using Razor Pages. Then select **Ok**. Be sure that **ASP.NET Core 2.1** is selected from the framework selector.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
+
+The libraries that contain SignalR server-side code are included in the project template. You must include the client side JavaScript file.
+
+1. Install the JavaScript client library with [npm](https://www.npmjs.com/). Open a command window at the root of your project and run `npm install`. `npm install` creates a *node_modules* folder and sub-folders for the libraries in the same location as where you run the npm command.
+
+  ```console
+   npm install @aspnet/signalr-client --save
+  ```
+
+1. Copy the *signalr.js* for SignalR from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
+
+## Create the SignalR Hub
+
+A hub is a class that represents a high-level pipeline that allows the client and server to call methods on each other directly.
+
+1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**. 
+
+1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data.
+
+1. Create the `Send` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous and scales better.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/ChatHub.cs?range=7-14)]
+
+## Configure the project to use SignalR
+
+The SignalR server must be configured so that it knows to pass requests to SignalR.
+
+1. To configure a SignalR project, modify the `ConfigureServices` method of the application's `Startup` class by inserting a call to `services.AddSignalR()`.
+
+  This adds SignalR as part of the [ASP.NET Core middleware](xref:fundamentals/middleware/index) pipeline.
+
+1. In the `Startup` class, you must also configure SignalR routes by calling `UseSignalR`. In the call to `UseSignalR` is where you setup the SignalR hub mappings.
```

Comment:
@anurse  can you here?
Do you config routes our setup hub mappings. Why do you need two sentences? I think it reads better something like
Configure SignalR routes and hub mappings by calling `UseSignalR`:
No need to tell them about Startup again. also is a banned word.

Reply:
I agree, it doesn't need to specified twice.

Reply:
Also, let's refer to them as "hub routes" (since we use the same routing system as MVC, which refers to them as routes.

Reply:
I think just "Configure routes to your hubs using `UseSignalR`". "Hub Mappings" are not really a thing. A Hub has a route that defines the URLs that map to it.